src/sentry/mediators/token_exchange/refresher.py

```diff
@@ -26,10 +27,24 @@ class Refresher(Mediator):
     user = Param(User)
     using = router.db_for_write(User)
 
+    @sentry_sdk.trace
     def call(self):
+        sentry_sdk.set_context(
+            "token-exchange.refresh",
+            {
+                "user_id": self.user.id,
+                "install_id": self.install.id,
+                "client_id": self.client_id,
+                "org_id": self.install.organization_id,
+                "sentry_app_id": self.sentry_app.id,
+                "sentry_app_slug": self.sentry_app.slug,
+                "application_id": self.application.id,
```

Comment:
Some of these fields should already be included when we capture an error, but I'm not sure which exact ones so all of them are here currently.

Reply:
Having both the sentry_app_id and sentry_app_slug is a bit redundant. Similarly the client_id is less important when you have `sentry_app_id` as you can use the id to find the client_id.